base/multi.jl

```diff
@@ -1544,13 +1544,13 @@ function pmap(f, lsts...; err_retry=true, err_stop=false, pids = workers())
     nextidx = 0
     getnextidx() = (nextidx += 1)
 
-    states = [start(lsts[idx]) for idx in 1:len]
+    states = [start(lst) for lst in lsts]
```

Comment:
Surprisingly, changing it to `map(start, lsts)` leads to an error in tests (see [log](https://ci.appveyor.com/project/JuliaLang/julia/build/1.0.209/job/n8xqmtnh43uvyd04)). I will change it to the original code, but it may uncover some other issue.